modules/commerce_bluesnap_currency/src/Form/CommerceBluesnapCurrencyResolverConversion.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Drupal\commerce_bluesnap_currency\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\commerce_currency_resolver\Form\CommerceCurrencyResolverConversion;
+
+/**
+ * Class CommerceCurrencyResolverConversion.
+ *
+ * @package Drupal\commerce_currency_resolver\Form
+ */
+class CommerceBluesnapCurrencyResolverConversion extends CommerceCurrencyResolverConversion {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    // Get current settings.
+    $config = $this->config('commerce_currency_resolver.currency_conversion');
+
+    // Call the parent form.
+    $form = parent::buildForm($form, $form_state);
+
+    // Unset the form submit coming from parent form as
+    // its been added below.
+    unset($form['submit']);
```

Comment:
@safalliajoseph Why are you removing and re-adding the submit button?

Reply:
The bluesnap settings form array was coming after the submit button as we call it after `parent::buildForm`. That is why I unset and recall it. 
Can we handle that using weights?